src/index.jsx

```diff
@@ -8,13 +8,12 @@ import 'normalize.css';
 import '../fonts/fonts.css';
 import './styles/globalStyles.css';
 
-import { sentryDsn } from './constants/apiKeys';
 import pjson from '../package.json';
 import App from './App';
 
 if (!__DEV__) {
   Sentry.init({
-    dsn: sentryDsn,
+    dsn: 'FIXME-sentryDsn',
```

Comment:
@Emily-Ke @Atticus29 I think it's not possible to get site settings here, what can I do instead?